src/libraries/pkg/test/frameworkSettings/net6.0/settings.targets

```diff
@@ -1,6 +1,8 @@
 <Project>
   <PropertyGroup>
     <RuntimeFrameworkVersion>$(MicrosoftNETCoreAppVersion)</RuntimeFrameworkVersion>
+    <!-- This should be enabled once we have updated assembly versions. -->
```

Comment:
Make sure we have an issue so we don't forget.

Reply:
Yeah we are tracking this here https://github.com/dotnet/runtime/issues/44078

Reply:
But this PR will close that issue 😄